build/index.html

```diff
@@ -0,0 +1,130 @@
+<html>
```

Comment:
Why is the build folder committed?

Reply:
Ignore it I was just checking responsiveness